x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/actions_menu.tsx

```diff
@@ -67,7 +67,7 @@ export const AgentDetailsActionMenu: React.FunctionComponent<{
       onClick={() => {
         setIsReassignFlyoutOpen(true);
       }}
-      disabled={!agent.active}
+      disabled={!agent.active || agentPolicy?.is_managed}
```

Comment:
I think the UX would be less confusing if we hid these options instead of disabling them

Reply:
Sure I will make the change